src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
We already did a check for Mendix 6 deprecations when I did [this commit](https://github.com/mendix/BooleanSlider/commit/68d9ec88906582f1ecebd7cc64d148b47d50590b). So this PR I am going to decline. What I will do is have a look at the deprecation-checker to see if I can eliminate this false-positive